### DISCUSSION TITLE
recently installed 2.4 can view storage filling up, services are online but on GUI unable to see any alerts, Grid says OK

### DISCUSSION THREAD

**Discussion by smallboy69** [Security-Onion-Solutions/securityonion#12092] [2.4]
### Version

2.4.30

### Installation Method

Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

20

### RAM

256gb

### Storage for /

9TB

### Storage for /nsm

9TB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

recently install Security Onion 2.4.30 i can see the storage, services online and GUI working but am not receiving any alerts in the interface Alerts

reinstalled the Security Onion and reboot system but to no avail, appreciate if you could please advise which logs to start checking why no alerts are showing on the GUI..

**Edited to removed docx. (@DefensiveDepth)

Thanks  

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
> Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.

First, please note that network installations are not officially supported:
https://docs.securityonion.net/en/2.4/os.html#supported

> recently install Security Onion 2.4.30 i can see the storage, services online and GUI working but am not receiving any alerts in the interface Alerts
reinstalled the Security Onion and reboot system but to no avail, appreciate if you could please advise which logs to start checking why no alerts are showing on the GUI..

Please see the Troubleshooting Alerts section of the documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

> **Edited to removed docx. (@defensivedepth)

Please review https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
_If you attach files, please make sure they are plain text format. No Word docs or PDFs please._

**Reply by smallboy69** [reply]
Thank you doug 

Dually noted and appreciative  for the advise and insight

Regards
